test/GithubApi.Gist.test.js

```diff
@@ -0,0 +1,100 @@
+const axios = require('axios');
+const { expect, assert } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+
+chai.use(chaiSubset);
+
+describe('Consume DELETE Method', () => {
+  const urlBase = 'https://api.github.com';
+  let createGistResponse;
+  let infoGistResponse;
+  let deleteGistResponse;
+  let deletedResponse;
+
+  describe('Create a gist', () => {
+    before(async () => {
+      createGistResponse = await axios.post(`${urlBase}/gists`, {
+        description: 'this is a gist example with a promise',
+        public: true,
+        files: {
+          'myPromise.js': {
+            content: `var myPromise = new Promise(function(resolve, reject) {
+                setTimeout(function() {
+                  resolve('Tick, Tick... Boom!');
+                }, 3000);
+              });`
+          }
+        }
+      }, {
```

Comment:
instead of writing 2 times this gist body, create a variable with this content and then reuse it, so this section can be:
```suggestion
      createGistResponse = await axios.post(`${urlBase}/gists`, gist, {
```